content/blog/posts/New-Project-April2017.rst

```diff
@@ -0,0 +1,13 @@
+New Project: New ftp Site for libpng Tarbell Distributions
+==========================================================
+:date: 2017-04-28
+:slug: new-project-ftp
+:author: Amanda Kelner
+:tag: featured-stories
+
+The OSL took on a new project and delivers a new ftp site for libpng tarbell
+distributions that is 20+ times as fast as the old site. It also now responds to
+http and https requests and ftp requests. For more information and links, see
```

Comment:
Are we providing this or are we maintaining the website which provides these things?